lib/js/styles/ItemsList.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+	<div>
+		<div v-for="itemsList in itemsLocal" :key="itemsList.title" class="itemDefinition">
+			<div class="itemsList" :class="{ [itemsList.class]: itemsList.class }">
+				<div class="itemsList__row">
+					<h2 class="itemsList__title">{{ itemsList.title }}</h2>
+					<ol class="itemsList__categories">
+						<li
+							v-for="(items, itemCategory) in itemsList.list"
+							:key="itemCategory"
+							class="itemDefinition__category"
+						>
+							{{ itemCategory }}
+						</li>
+					</ol>
+				</div>
+				<div
+					v-for="(items, itemCategory) in itemsList.list"
+					:key="itemCategory"
+					class="itemsList__row"
+				>
+					<h3 :id="'category-' + itemCategory">{{ itemCategory }}</h3>
+					<item-color v-if="type === TOKENS_TYPES.COLORS" :items="items" />
+
+					<!-- @todo replace content for ItemTypography -->
```

Comment:
don't forget

Reply:
I will touch it again later, don't close this thread.

Reply:
Removed.